ocdaction/ocdaction/templates/challenge/challenge_list.html

```diff
@@ -1,48 +1,94 @@
+                            <table class="table table-striped">
+                                <thead>
+                                <tr>
+                                    <th>Level</th>
+                                    <th>Challenge</th>
+                                    <th></th>
+                                </tr>
+                                </thead>
+                                <tbody>
+                                    {% for challenge in challenges_in_progess %}
+                                        <tr>
+                                            <td class="text-align-left ibox-content">{% get_anxiety_level challenge %}</td>
+                                            <td class="text-align-left ibox-content"><a href="{% url 'challenge' challenge.id %}">{{ challenge.challenge_name }}</a></td>
+                                            <td class="text-align-right ibox-content"><a class="" href="">Continue</a></td>
+                                        </tr>
+                                    {% endfor %}
+                                </tbody>
+                            </table>
+                        </div>
+                    </div>
+                </div>
             {% endif %}
-        {% empty %}
-            <ul class="col-xs-6 col-xs-offset-3">
-                <h2>You have no challenges yet!</h2>
-            </ul>
-        {% endfor %}
+
+            {% if challenges_not_in_progress %}
+                <div class="col-lg-12">
+                    <div class="ibox float-e-margins">
+                        <div class="ibox-title">
+                            <h5 class="h3 text-align-center">My Challenges</h5>
+                            <div class="ibox-tools">
+                                <a class="close-link">
+                                    <i class="fa fa-times"></i>
+                                </a>
+                            </div>
+                        </div>
+                        <div class="ibox-content">
+                            <table class="table table-striped">
+                                <thead>
+                                <tr>
+                                    <th>Level</th>
+                                    <th>Challenge</th>
+                                    <th></th>
+                                </tr>
+                                </thead>
+                                <tbody>
+                                    {% for challenge in challenges_not_in_progress %}
+                                        <tr>
+                                            <td class="text-align-left">{% get_anxiety_level challenge %}</td>
+                                            <td class="text-align-left"><a href="{% url 'challenge' challenge.id %}">{{ challenge.challenge_name }}</a></td>
+                                            <td><a class="" href="">Do challenge</a></td>
```

Comment:
We don't use 'Do the challenge' (or 'task') anymore as it doesn't reflect the exposure cycle. Just an arrow, like in the design, will be enough.